### ISSUE TITLE
Naming/VariableNumber triggers on `:TLS1_2` parameter to `Net::HTTP.start`

### ISSUE THREAD

**Issue by Zopolis4** [rubocop/rubocop#13977]
## Expected behavior

I expected rubocop to not raise an offense on one of the parameters to a standard library gem. 

## Actual behavior

Rubocop raised an offense on `:TLS1_2`, which is one of the accepted options listed here: https://docs.ruby-lang.org/en/master/OpenSSL/SSL/SSLContext.html#method-i-min_version-3D

## Steps to reproduce the problem

example.rb:
```ruby
Net::HTTP.start(uri.host, uri.port, { min_version: :TLS1_2 })
```

```
$ rubocop --only Naming/VariableNumber example.rb

Inspecting 1 file
C

Offenses:

example.rb:1:52: C: Naming/VariableNumber: Use normalcase for symbol numbers.
Net::HTTP.start(uri.host, uri.port, { min_version: :TLS1_2 })
                                                   ^^^^^^^

1 file inspected, 1 offense detected
```

## RuboCop version

```
1.73.2 (using Parser 3.3.7.1, rubocop-ast 1.38.1, analyzing as Ruby 3.4, running on ruby 3.4.2) [x86_64-linux-gnu]
```

**Comment by koic** [comment]
This issue has been resolved in #13978.
